frontend/pendingTranslations.ftl

```diff
@@ -90,20 +90,20 @@ phone-onboarding-step4-button-confirm-relay-number = Confirm
 phone-onboarding-step4-code-success-title = Congratulations!
 phone-onboarding-step4-code-success-body = You’ve registered your new phone number mask.
 phone-onboarding-step4-code-success-subhead-title = What’s next?
-phone-onboarding-step4-code-success-subhead-body-p1 = { -brand-name-relay } sent you an SMS with a new contact card through which we’ll forward your calls and messages.
+phone-onboarding-step4-code-success-subhead-body-p1 = { -brand-name-relay } sent you an text with a new contact card through which we’ll forward your calls and messages.
 phone-onboarding-step4-code-success-subhead-body-p2 = Please save the contact so you can identify your forwarded messages and calls.
 phone-onboarding-step4-code-success-cta = Continue
 phone-onboarding-step4-results= No results found. Please try again.
 
 # Phone Settings
-phone-settings-caller-sms-log = Caller and SMS log
+phone-settings-caller-sms-log = Caller and text log
 phone-settings-caller-sms-log-description = Allow { -brand-name-firefox-relay } to keep a log of your callers and text senders.
-phone-settings-caller-sms-log-warning = If you decide to opt out from this preference, you will lose the ability to block and reply to senders or callers. If you’ve blocked any, they will become unblocked and your existing caller and SMS sender log will be permanently cleared from your history.
+phone-settings-caller-sms-log-warning = If you decide to opt out from this preference, you will lose the ability to block and reply to senders or callers. If you’ve blocked any, they will become unblocked and your existing caller and text sender log will be permanently cleared from your history.
 
 # Phone Resend SMS Banner
-phone-banner-resend-welcome-sms-cta = Resend welcome SMS
+phone-banner-resend-welcome-sms-cta = Resend welcome text
 phone-banner-resend-welcome-sms-title = Quick Tip
-phone-banner-resend-welcome-sms-body =  Remember to save the contact we shared with you by SMS to help you identify messages forwarded by { -brand-name-relay }. Can’t find it?
+phone-banner-resend-welcome-sms-body =  Remember to save the contact we shared with you by text to help you identify messages forwarded by { -brand-name-relay }. Can’t find it?
```

Comment:
Might be my not being a native speaker, but this gets pretty ambiguous for me - shouldn't this be "text message"? (I had the same feeling for "Caller and text log" above.)

Reply:
I think 'by text' reads fine since it implies a message. Good callout on the "Caller and text log" above, I checked the design specs again and it was actually Caller and _texts_ log, which sounds better.